app/src/main/scala/org/alephium/explorer/persistence/schema/InputSchema.scala

```diff
@@ -49,6 +49,8 @@ object InputSchema extends SchemaMainChain[InputEntity]("inputs") {
     def outputRefKeyIdx: Index     = index("inputs_output_ref_key_idx", outputRefKey)
     def timestampIdx: Index        = index("inputs_timestamp_idx", timestamp)
     def outputRefAddressIdx: Index = index("inputs_output_ref_address_idx", outputRefAddress)
+    def inputsBlockHashTxHashIdx: Index =
+      index("inputs_block_hash_tx_hash_idx", (blockHash, txHash))
```

Comment:
`txHash` should come before `blockHash`, and let's make sure that query planner uses `txHash` index first.

Reply:
I swap them all so it's coherent everywhere. Here is the planners:

```
explorer=# EXPLAIN SELECT tx_hash, input_order, hint, output_ref_key, unlock_script, output_ref_address, output_ref_amount, output_ref_tokens
explorer-# FROM inputs
explorer-# WHERE (tx_hash, block_hash) IN (('arst','art'));
                                          QUERY PLAN
----------------------------------------------------------------------------------------------
 Index Scan using inputs_tx_hash_block_hash_idx on inputs  (cost=0.43..8.45 rows=1 width=232)
   Index Cond: ((tx_hash = '\x61727374'::bytea) AND (block_hash = '\x617274'::bytea))
(2 rows)

explorer=# EXPLAIN SELECT outputs.tx_hash, outputs.output_order, outputs.output_type, outputs.hint, outputs.key, outputs.amount, outputs.address, outputs.tokens, outputs.lock_time, outputs.message, outputs.spent_finalized
explorer-# FROM outputs
explorer-# WHERE (outputs.tx_hash, outputs.block_hash) IN (('arst','art'));
                                           QUERY PLAN
------------------------------------------------------------------------------------------------
 Index Scan using outputs_tx_hash_block_hash_idx on outputs  (cost=0.55..8.57 rows=1 width=217)
   Index Cond: ((tx_hash = '\x61727374'::bytea) AND (block_hash = '\x617274'::bytea))
(2 rows)

explorer=# EXPLAIN SELECT hash, gas_amount, gas_price
explorer-# FROM transactions
explorer-# WHERE (hash, block_hash) IN (('arst','art'));
                                    QUERY PLAN
----------------------------------------------------------------------------------
 Index Scan using txs_hash_idx on transactions  (cost=0.43..8.45 rows=1 width=42)
   Index Cond: (hash = '\x61727374'::bytea)
   Filter: (block_hash = '\x617274'::bytea)
(3 rows)
```

so both inputs and outputs queries are using the index, but not the query for the gas, maybe postgres can optimize an other way.

Reply:
Perfect 👍